File: Car_PC/CarInfo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Car_PC
{
    public  class CarInfo
    {
        public CarInfo(string data)
        {
            string[] datas = data.Split(',');
            this.Engine1 = int.Parse(datas[0]);
            this.Engine2 = int.Parse(datas[1]);
            this.Engine3 = int.Parse(datas[2]);
            this.Engine4 = int.Parse(datas[3]);

            this.LeftSpeed = int.Parse(datas[4]);
            this.RightSpeed = int.Parse(datas[5]);

            this.ServoX = int.Parse(datas[6]);
            this.ServoY = int.Parse(datas[7]);

            this.RadarStatus = int.Parse(datas[8])>0?true:false;
            this.LightStatus = int.Parse(datas[9]) > 0 ? true : false;
        }
        public  int Engine1 { get; set; }
        public  int Engine2 { get; set; }
        public  int Engine3 { get; set; }
        public  int Engine4 { get; set; }

        public int LeftSpeed { get; set; }
        public int RightSpeed { get; set; }
        public int ServoX { get; set; }
        public int ServoY { get; set; }

        public bool RadarStatus { get; set; }
        public bool LightStatus { get; set; }
    }
}

File: Smart Car/app/src/main/java/com/iha/wcc/job/car/CarInfo.java
package com.iha.wcc.job.car;

/**
 * Created by Administrator on 2015/6/17.
 */
public  class CarInfo
{
    public CarInfo(String data)
    {
        String[] datas = data.split(",");

        this.Engine1 = Integer.parseInt(datas[0]);
        this.Engine2 = Integer.parseInt(datas[1]);
        this.Engine3 = Integer.parseInt(datas[2]);
        this.Engine4 = Integer.parseInt(datas[3]);

        this.LeftSpeed = Integer.parseInt(datas[4]);
        this.RightSpeed = Integer.parseInt(datas[5]);

        this.ServoX = Integer.parseInt(datas[6]);
        this.ServoY = Integer.parseInt(datas[7]);

        this.RadarStatus =Integer.parseInt(datas[8])>0?true:false;
        this.LightStatus = Integer.parseInt(datas[9]) > 0 ? true : false;
    }
    public  int Engine1 ;
    public  int Engine2 ;
    public  int Engine3;
    public  int Engine4 ;

    public int LeftSpeed ;
    public int RightSpeed;
    public int ServoX;
    public int ServoY;

    public boolean RadarStatus;
    public boolean LightStatus ;
}
